optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -858,9 +813,6 @@ def tune_params(self, target_param_names, n_trials, sampler, step_name):
             target_param_names, n_trials, sampler, step_name
         )
 
-        # Add tuning history.
-        self.tuning_history += objective.report
```

Comment:
I think `OptunaObjective.report` is no longer used and can be also deleted.

Reply:
Thanks, you're right. Removed `OptunaObjective.report`.